Blockzilla/AddCustomDomainViewController.swift

```diff
@@ -96,7 +96,7 @@ class AddCustomDomainViewController: UIViewController, UITextFieldDelegate {
             guard !error.message.isEmpty else { return }
             Toast(text: error.message).show()
         case .success:
-            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.customDomainAdded, object: TelemetryEventObject.setting)
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.change, object: TelemetryEventObject.customDomain)
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet. @boek is this OK with you?